vassal-app/src/main/java/VASSAL/counters/FreeRotator.java

```diff
@@ -126,7 +126,7 @@
 
   public FreeRotator() {
     // modified for random rotation (added two ; )
-    this(ID + "6;];[;" + Resources.getString("Editor.FreeRotator.default_rotate_cw_command") + ";" + Resources.getString("Editor.FreeRotator.default_rotate_ccw_command") + ";;;;", null); //$NON-NLS-1$//
+    this(ID + "6;];[;" + Resources.getString("Editor.FreeRotator.default_rotate_cw_command") + ";" + Resources.getString("Editor.FreeRotator.default_rotate_ccw_command") + ";;;;;", null); //$NON-NLS-1$//
```

Comment:
The extra ';' should no longer be needed as we are no longer adding the start  angle option?